src/Elastic.Apm/Report/Serialization/BodyStringSanitizerConverter.cs

```diff
@@ -59,6 +59,9 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 		/// <returns></returns>
 		private static bool FormatCheck(string bodyValue)
 		{
+			if (!bodyValue.Contains("=") && !bodyValue.Contains("&"))
```

Comment:
IMHO this check introduces a regression - after this change `"password=qwerty"` will not be sanitized because it doesn't contain `&`.

Reply:
On it

Reply:
I don't think so 🤔 
There is an `&&` between to two `bodyValue.Contains` checks, so it's enough to have a `=` in it to not return. I pushed 2 new tests, one tests against your scenario - seems working and the code also makes sense to me.

Reply:
You are right - my mistake. Do we have tests that invoke sanitization and serialization together (as they are invoked in production code) to verify serialization doesn't neutralize sanitization by mistake?